README.md

```diff
@@ -914,6 +914,19 @@ nt-dates/data/](https://amiibereval2018.wordpress.com/important-dates/data/)
 * Medium: Text
 * Reference: Salminen, J., Almerekhi, H., Milenković, M., Jung, S., An, J., Kwak, H. and Jansen, B., 2018, Anatomy of Online Hate: Developing a Taxonomy and Machine Learning Models for Identifying and Classifying Hate in Online News Media, In: Proceedings of the Twelfth International AAAI Conference on Web and Social Media (ICWSM 2018), pp.330-339
 
+#### HateXplain: A Benchmark Dataset for Explainable Hate Speech Detection 
+* Link to publication: [https://arxiv.org/pdf/2012.10289.pdf](https://arxiv.org/pdf/2012.10289.pdf)  
+* Link to data: [https://github.com/punyajoy/HateXplain](https://github.com/punyajoy/HateXplain)  
+* Task description: Binary (Hate, Not) and Three-class (Hate speech, Offensive language, None) 
+* Details of task: Hatespeech detection on social media in English, including 10 categories: African, Islam, Jewish, LGBTQ, Women, Refugee, Arab, Caucasian, Hispanic, Asian 
+* Size of dataset: 20148 
+* Percentage abusive: 0.57 
```

Comment:
Is this 57% or 0.57% ? Can you add a percentage sign to clarify the number?

Reply:
It is 57%. I will send a new pull request.